app/views/layout.html

```diff
@@ -18,4 +18,5 @@
 
 {% block body_end %}
   {% include "includes/scripts.html" %}
+  <!-- GOV.UK Prototype kit {{releaseVersion}} -->
```

Comment:
We could, but the head is automatically collapsed when you open the inspector (chrome for mac at least) - so I slightly prefer the footer.